presto-testing/src/main/java/io/prestosql/testing/AbstractTestIntegrationSmokeTest.java

```diff
@@ -33,12 +33,16 @@
 import static org.testng.Assert.assertTrue;
 
 /**
+ * Any new test added here should also be added to {@link BaseConnectorTest}.
```

Comment:
The first sentence of the javadoc should tell what the class is, so please do not add this here.
I think it's OK not to add this sentence at all, deprecation should be enough.